docs/pbc/all/merchant-management/202407.0/marketplace/marketplace-merchant-commission-feature-overview.md

```diff
@@ -0,0 +1,192 @@
+
+You might want to set up different commission rules based on this since it affects commission totals.
+
+For example, if customer uses Net mode, you might want to apply 10% commission, while if customer uses Gross mode commission applied will be 12%.
+
+
+
+
+### Combining conditions
+
+Conditions can be combined using `AND` and `OR` operators. Examples:
+
+* `category IS IN 'electronics';smart-watches' AND attribute.color IS IN 'black;blue'`
+* `category IS IN 'electronics';smart-watches' OR attribute.color IS IN 'black;blue'`
+
+When conditions are combined by the `OR` operator, they don't exclude each other. If an order fulfills both such conditions, the commission is still applied.
+
+
+
+### Merchant-based conditions
+
+
+The `merchants_allow_list` attribute lets you apply a commission to one or more particular merchants. This is useful when you only want to apply a commission to some of your merchants; or when you have a universal commission for most merchants and a special commission for some of them. For example, your setup could be as follows. We simplified the commissions for this example by removing irrelevant fields.
+
+| KEY | PRIORITY | AMOUNT | merchants_allow_list |
+| mc01 | 1 | 10 | MER000002,MER000004 |
+| mc02 | 2 | 5 |   |
+
+In the prior example, the system first checks if an order is fulfilled by merchants `MER000002` or `MER000004`. If this condition is fulfilled, the `mc01` commission is applied. If an order is fulfilled by any other merchant, the universal commission `mc02` is applied. The empty `merchants_allow_list` attribute means that this commission applies to all merchants.
+
+If you have multiple commissions, depending on your setup, you will have to use priorities and groups to make sure that relevant commissions are applied. In the prior example, the merchant-specific commission with priority `1` is validated before the universal commission. If the merchant-specific commission had a lower priority, the universal commission would be applied because it doesn't have a merchant condition. For more details about priority, see []
+
+
+
+
+## Merchant commission priority and groups
+
+Priority and groups are used to make sure relevant commissions are applied when there are multiple commissions in a system.
+
+Merchant commission groups are used when multiple commissions need to be applied to per order or order item. By default, there are `primary` and `secondary` commission groups. When commissions of both groups exist in a shop, for each order or order item, one commission from each group is applied; the commission from the primary group is applied first.
```

Comment:
We used alot term "apply commission to order or order item", while condition exist for order level, we technically can apply commission only on item level, not on order level. Lets rephrase that or remove it from the document.